doc/dev_guide/release_management.rst

```diff
@@ -115,6 +115,14 @@ channel (`tarantool_news <https://t.me/tarantool_news>`_).
 We also publish nightly builds, and use the fourth slot in the version
 identifier to designate the nightly build number.
 
+.. important::
+
+   Only a version with ``0`` in the fourth slot, namely ``X.Y.Z-0-<hash>`` is
```

Comment:
`X.Y.Z-0-g<hash>` to be precise. 'g' is 'git' I guess.

Reply:
corrected